src/components/Settings/SettingsNotifications/NotificationModal/GotifyModal.tsx

```
@@ -0,0 +1,259 @@
+  data,
+  onClose,
+  onTest,
+  onSave,
+}: GotifyModalProps) => {
+  const intl = useIntl();
+
+  const NotificationsGotifySchema = Yup.object().shape({
+    url: Yup.string()
+      .when('enabled', {
+        is: true,
+        then: Yup.string()
```

Biome (2.4.4)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)